src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Handlers/ProjectPropertiesItemHandler.cs

```diff
@@ -27,6 +40,9 @@ public void Handle(IComparable version, IProjectChangeDescription projectChange,
             Requires.NotNull(projectChange, nameof(projectChange));
             Requires.NotNull(logger, nameof(logger));
 
+            if (_context == null)
+                throw new InvalidOperationException();
```

Comment:
Can we make this error more descriptive? How does someone immediately know that this is because the class wasn't initialized properly?